### ISSUE TITLE
Failing dotnet database update

### ISSUE THREAD

**Issue by SimonRuehle** [dotnet/cli#4072]
## Steps to reproduce
Difficult to tell, on some PCs the error is reproducable on others not.
Tested the behaviour with LocalDB and SQL Server Express.

## Expected  behavior
After a dotnet database drop (-f) the database is deleted, if it did not exist nothing changes
After dotnet database update a new one is created.

## Actual behavior
dotnet database drop cannot delete database,
after manual deletion cannot delete because not exists.
Then dotnet database update fails because database exists, even if database drop reports database does not exist.
On file system the database does not exist (in case of LocalDB).
So in short drop fails because not exists, update because exists.

## Environment data
`dotnet --info` output:
.NET Command Line Tools (1.0.0-preview2-003121)

Product Information:
 Version:            1.0.0-preview2-003121
 Commit SHA-1 hash:  1e9d529bc5

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64

**Comment by svick** [comment]
Are you talking about [the `dotnet ef database` commands](https://docs.efproject.net/en/latest/miscellaneous/cli/dotnet.html#dotnet-ef-database)? In that case, I think this should be reported on [the aspnet/EntityFramework repo](https://github.com/aspnet/EntityFramework).

**Comment by SimonRuehle** [comment]
You are correct, I am sorry for this, but I cannot close the issue.
